test/lando.spec.js

```diff
@@ -45,15 +45,19 @@ describe('lando', () => {
   });
 
   describe('#bootstrap', () => {
-    it('should return a lando object with the default config', () => {
-      const lando = new Lando({logLevelConsole: 'warn'});
+    it('should return a lando object with the default config', function() {
+      // eslint-disable-next-line no-invalid-this
+      this.timeout(45000);
```

Comment:
Tests time out without this on Windows, and I cannot figure out why :-(

Reply:
the best type of comment :)

Reply:
during my testing on Windows I wasn't able to replicate this. There weren't any discernible delays.